newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -67,9 +71,20 @@ export default function SummaryPage() {
   }
 
   const mailSent = sendMailResponse !== null;
-
   return (
     <Container text>
+      <FormGroup row>
+        <FormControlLabel
+          control={
+            <Switch
+              checked={checked}
+              onChange={event => setChecked(event.target.checked)}
+              name="toggle"
+            />
+          }
+          label="Toggle view"
```

Comment:
The grid looks a little weird if nobody voted yet because then it only consists of the headers. Maybe it makes sense to only be able to switch when at least one person has voted already (or has been invited, just so that the table isn't completely empty)?